### PR TITLE
Release Notes 7-31/2022-8-7-2022

### PR DESCRIPTION
**Pull request by miwithro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Konnectivity rollout is finished in global and started in Sovereign (China, USGov).
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+### Release notes
+
+* Features
+  * GA of Kubernetes 1.24
+  * Deprecation of Kubernetes 1.21
```
Comment (palma21) [inline CHANGELOG.md:28]: Not really a feature. Behavioral change.

Kubernetes 1.21 is now _removed_.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+* Behavioral Changes
+  * increased memory request (20Mi -> 40Mi) for azuredisk and node-driver-registrar containers in azurediskcsi-azuredisk-v2-node
```
Comment (palma21) [inline CHANGELOG.md:30]: Capitalize I

Comment (miwithro) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (miwithro) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
 
+## Release 2022-08-07
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * The host Linux VM operating system will be Ubuntu 22.04 for Intel and ARM64 architectures
+  * The host Windows VM operating system will be Windows Server 2022
```
Comment (palma21) [inline CHANGELOG.md:11]: *Default host?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-08-07
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * The host Linux VM operating system will be Ubuntu 22.04 for Intel and ARM64 architectures
```
Comment (palma21) [inline CHANGELOG.md:10]: x86 (not intel, we also support AMD)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,39 @@
...
+  * The host Windows VM operating system will be Windows Server 2022
+
```
Comment (palma21) [inline CHANGELOG.md:12]: Extra line

**Review by palma21 — APPROVED** [review]

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,38 @@
...
+* Features
+  * GA of Kubernetes 1.24
+* Behavioral Changes
+  * Deprecation of Kubernetes 1.21
+  * Increased memory request (20Mi -> 40Mi) for azuredisk and node-driver-registrar containers in azurediskcsi-azuredisk-v2-node
+* Component Updates
+  * Calico is updated to v3.21.6
+  * CSI Secret Store now supports Windows Server 2022
+  * Microsoft Defender for Containers images updated 1.0.67
```
Comment (shashankbarsin) [inline CHANGELOG.md:33]: nit: 'updated to'. is there any release notes for this image we can link to?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,38 @@
...
+
+### Release notes
+
+* Features
+  * GA of Kubernetes 1.24
+* Behavioral Changes
+  * Deprecation of Kubernetes 1.21
+  * Increased memory request (20Mi -> 40Mi) for azuredisk and node-driver-registrar containers in azurediskcsi-azuredisk-v2-node
+* Component Updates
+  * Calico is updated to v3.21.6
```
Comment (shashankbarsin) [inline CHANGELOG.md:31]: can we add link to this release - https://github.com/projectcalico/calico/releases/tag/v3.21.6

File: CHANGELOG.md (hunk 6 above)
Comment (palma21) [reply CHANGELOG.md:33]: https://projectcalico.docs.tigera.io/archive/v3.21/release-notes/

And 3.21.5 (is actually the reason for this update) contains an important fix for AKS specifically that would be important to call out. Thanks for the reminder.

File: CHANGELOG.md (hunk 7 above)
Comment (palma21) [reply CHANGELOG.md:31]: use the one above instead due to the reason I mentioned

**Review by palma21 — CHANGES_REQUESTED** [review]
comments in-line

File: CHANGELOG.md (hunk 7 above)
Comment (miwithro) [reply CHANGELOG.md:31]: already added ^^

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,38 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-08-07
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
```
Comment (palma21) [inline CHANGELOG.md:10]: _linux host_
Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows 2022 hosts.

**Review by palma21 — APPROVED** [review]

**Review by shashankbarsin — APPROVED** [review]

**Pull request merged**